vue/sbc-common-components/src/util/constants.ts

```diff
@@ -55,3 +55,5 @@ export enum Pages {
 }
 
 export const ALLOWED_URIS_FOR_PENDING_ORGS: string[] = ['setup-non-bcsc-account']
+
+export const ACCOUNT_ID = 'accountid'
```

Comment:
I thought you said it was going to be "accountId" ?

Reply:
since its URL, i think it's better to go with small letter  or -